src/Jira/Api/Client/PHPClient.php

```diff
@@ -201,11 +201,13 @@ protected function doSendRequest($url, array $context)
 	{
 		$this->_lastErrorMessage = '';
 
-		set_error_handler(array($this, 'errorHandler'));
+		/** @var callable $callable */
+		$callable = array($this, 'errorHandler');
+		set_error_handler($callable);
 		$response = file_get_contents($url, false, stream_context_create($context));
 		restore_error_handler();
 
-		if ( isset($http_response_header) ) {
+		if ( $http_response_header ) {
```

Comment:
The changed code version expects, that `$http_response_header` would always be defined. Is this the case?

**This change should be covered by tests.**

Reply:
PHPStan knows this can never be null, only empty.

Reply:
Oh, I wouldn't blindly trust any static code analyzer. Please back this change with the corresponding tests.

Reply:
I do in this case. Those are correct, if you also look into source code and PHP docs.
The code itself doesnt make sense as it was.

Reply:
I admire the energy you're putting into defending the static code analyzer. I would love to see that redirected into test writing. :) Not joking.